src/Nethermind/Nethermind.JsonRpc/JsonRpcProcessor.cs

```diff
@@ -124,9 +126,18 @@ public async IAsyncEnumerable<JsonRpcResult> ProcessAsync(PipeReader reader, Jso
 
         // Handles general exceptions during parsing and validation.
         // Sends an error response and stops the stopwatch.
-        JsonRpcResult GetParsingError(string error, Exception? exception = null)
+        JsonRpcResult GetParsingError(ReadOnlySequence<byte> buffer, string error, Exception? exception = null)
```

Comment:
pass it by `ref`?

Reply:
Is it needed? It's just 2 pointers + 2 ints: 
![image](https://github.com/user-attachments/assets/293c589f-9719-49d5-b76f-445e4b069832)

Reply:
A bit 8 bytes vs 24 bytes.

Reply:
Made it pass via `in`/`readonly ref`  everywhere, but I do have some doubts this will provide any performance benefits.